toolboxes/dwt/cpu/hoNDHarrWavelet.cpp

```diff
@@ -0,0 +1,588 @@
+
+#include "hoNDHarrWavelet.h"
+
+namespace Gadgetron{
+
+template<typename T> 
+hoNDHarrWavelet<T>::hoNDHarrWavelet()
+{
+}
+
+template<typename T>
+hoNDHarrWavelet<T>::~hoNDHarrWavelet()
+{
+}
+
+template<typename T>
+void hoNDHarrWavelet<T>::dwt1D(const T* const in, T* out, size_t RO, size_t level)
+{
+    try
```

Comment:
the try catch block here does nothing, as none of the operations can throw. They can segfault, but not throw.